github/teams.go

```diff
@@ -476,3 +476,88 @@ func (s *TeamsService) RemoveTeamProject(ctx context.Context, teamID int64, proj
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GroupObject reprents list of Idp Group.
+type GroupObject struct {
+	Groups []*Group `json:"groups,omitempty"`
+}
+
+// Group represents an Idp group status.
+type Group struct {
+	GroupID          *string `json:"group_id,omitempty"`
+	GroupName        *string `json:"group_name,omitempty"`
+	GroupDescription *string `json:"group_description,omitempty"`
+}
+
+// ListIDPGroupsInOrganization list IdP groups available in an organization.
+//
+// Github API docs: https://developer.github.com/v3/teams/team_sync/
```

Comment:
Please use the official capitalization for `GitHub`.

URL should be: https://developer.github.com/v3/teams/team_sync/#list-idp-groups-in-an-organization